images/docker-compose.yml.in

```diff
@@ -52,6 +52,18 @@ services:
         networks:
             - eve
 
+    edgeview:
+        image: EDGEVIEW_TAG
+        privileged: true
+        volumes:
+            - persist:/persist:ro
+            - config:/config:ro
+            - run:/run:ro
+            - etc:/etc:ro
+            - proc:/host/proc:ro
+        network_mode: host
+        #networks:
```

Comment:
might as well remove this commented line.

Reply:
sure.